config/default/kustomization.yaml

```diff
@@ -0,0 +1,71 @@
+# Adds namespace to all resources.
+namespace: devworkspace-operator
```

Comment:
What I did wrong :
```bash
~/projects/devworkspace-operator update-sdk-1.x *3 !1 > make deploy                                                                                           9s % 14:44:25
kubectl create namespace devworkspace-controller || true
Error from server (AlreadyExists): namespaces "devworkspace-controller" already exists
kubectl apply -f config/registry/local -n devworkspace-controller
deployment.apps/che-plugin-registry unchanged
service/che-plugin-registry unchanged
kubectl apply -f config/registry/local/os -n devworkspace-controller
route.route.openshift.io/che-plugin-registry unchanged
mv config/devel/config.properties config/devel/config.properties.bak
mv config/devel/manager_image_patch.yaml config/devel/manager_image_patch.yaml.bak
envsubst < config/devel/config.properties.bak > config/devel/config.properties
envsubst < config/devel/manager_image_patch.yaml.bak > config/devel/manager_image_patch.yaml
/home/sleshche/gocode/bin/kustomize build config/devel | kubectl apply -f - || true
namespace/devworkspace-controller-system created
customresourcedefinition.apiextensions.k8s.io/components.controller.devfile.io configured
customresourcedefinition.apiextensions.k8s.io/devworkspaces.workspace.devfile.io configured
customresourcedefinition.apiextensions.k8s.io/workspaceroutings.controller.devfile.io configured
clusterrole.rbac.authorization.k8s.io/devworkspace-controller-edit-workspaces created
clusterrole.rbac.authorization.k8s.io/devworkspace-controller-manager-role created
clusterrole.rbac.authorization.k8s.io/devworkspace-controller-proxy-role created
clusterrole.rbac.authorization.k8s.io/devworkspace-controller-view-workspaces created
clusterrole.rbac.authorization.k8s.io/devworkspace-controller-metrics-reader created
clusterrolebinding.rbac.authorization.k8s.io/devworkspace-controller-manager-rolebinding created
clusterrolebinding.rbac.authorization.k8s.io/devworkspace-controller-proxy-rolebinding created
configmap/devworkspace-controller created
Error from server (NotFound): error when creating "STDIN": namespaces "devworkspace-operator" not found
Error from server (NotFound): error when creating "STDIN": namespaces "devworkspace-operator" not found
Error from server (NotFound): error when creating "STDIN": namespaces "devworkspace-operator" not found
Error from server (NotFound): error when creating "STDIN": namespaces "system" not found
mv config/devel/config.properties.bak config/devel/config.properties

```

Reply:
If this is on `crc` I suspect this might be me not making the project correctly. I'll double check.

Reply:
My bad -- I forgot to allow overriding namespace in the `config/devel` kustomization. Should be fixed now.